### PR TITLE
Feature/spp 196/snapshot

### PR DESCRIPTION
**Pull request by RyanAldy**

### CODE REVIEW

File: takeon-dbexport/main.go [hunk 1]
```diff
@@ -1,58 +1,110 @@
...
+	fmt.Println("Data from BL: " + string(dataFromBL))
 	if err != nil {
 		fmt.Printf("The HTTP request failed with error %s\n", err)
+		sendToSqs(snapshotID, "null", false)
 	} else {
 		data, _ := ioutil.ReadAll(response.Body)
-		cdbExport <- string(data)
+		cdbExport := string(data)
+		fmt.Println("Data from BL: " + cdbExport)
+		if strings.Contains(cdbExport, "Error loading data for db Export") {
```
Comment (sivagudur) [inline takeon-dbexport/main.go:96]: This Bit we need to handle it correctly.  What's happening is on Error is it is continuing the process and creating an empty file in S3 bucket and send a success 'true' flag to output queue. It was supposed to not to create a file and send location=null and success=false to outputqueue

**Review by sivagudur — CHANGES_REQUESTED** [review]
Need changes in handling ''Error" string returning from Business layer

**Review by mattprice-28 — COMMENTED** [review]
Quick review, pulled down and run unit tests locally, all passing, can see the lambda in aws and expected behaviour in cloudwatch, in cloudwatch I can see the message 'Data from BL:' three times, the first time it has the expected data after the colon but the other two are blank

**Review by mattprice-28 — APPROVED** [review]
Tested again, looks better now, just the 'Data from bl inside else' statement which I assume was there for testing

**Review by sivagudur — APPROVED** [review]
All looks good to me and it is working as expected based on confluence document. I tested both success 'true' and 'false' scenarios.

**Pull request merged**